enterprise/views.py

```diff
@@ -286,6 +290,12 @@ def get_account_consent(self, request):
             ).format(
                 enterprise_customer_name=customer.name
             ),
+            'confirmation_alert_prompt_warning': _(
+                'If you do not consent to share your course data, that information may be shared with '
```

Comment:
This is sort of an odd way to phrase the fact that we _may_ share the fact that the learner declined to give their consent to share their data with the enterprise customer -- I'm assuming this is the message that the legal team wants to present, though -- yes?